schema/output.js

```diff
@@ -1,43 +1,62 @@
 const Joi = require('joi');
 
 const metadataSchema = Joi.object({
-    legId: Joi.string().required()
+    legId: Joi.string().required(),
+    networkMos: Joi.number().positive().allow(0).required()
```

Comment:
I think maybe the mos calculation should occur in this package instead of on the client rtcstats_analytics.js file. What do you think? To me that makes it more consistent if we use this repository in mobile and on the backend, then we can be sure mos is always calculated the same way